dali/kernels/audio/mel_scale/mel_filter_bank_gpu.cu

```diff
@@ -193,8 +451,7 @@ class MelFilterBankGpu<T>::Impl : public MelFilterImplBase<T> {
       for (int64_t s = 0; s < nframes_.back(); ++s) {
         auto nblocks = div_ceil(nwindows_.back(), kBlockDim2);
         for (int64_t b = 0; b < nblocks; ++b) {
-          block_descs_.push_back(BlockDesc<T>{nullptr, nullptr,
-                                              {b * kBlockDim2, nwindows_.back()}});
+          block_descs_.emplace_back(0, 0, b * kBlockDim2, nwindows_.back());
```

Comment:
why changing `nullptr` with `0`?

Reply:
Because I've changed the constructor, now it takes offsets.